repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -578,18 +577,24 @@ export const clickRandomElementInTable = (tableName: string, elementId: string):
 };
 
 export const completeOperatorSearch = (isMultiService: boolean): void => {
-
     clickElementById('test-radio');
     continueButtonClick();
-
 };
 
 export const addFlatFareProductIfNotPresent = (): void => {
     getHomePage();
     clickElementById('manage-fares-link');
     clickElementByText('Other products');
-    getElementByClass('govuk-table').then((table) => {
-        if (table.find('tr').length === 1) {
+    const hasFlatFare: JQuery<HTMLElement>[] = [];
+    cy.wrap(hasFlatFare).as('hasFlatFare');
+    cy.get(`[id^="type-"]`).each(($element) => {
+        if ($element.text() === 'Flat fare') {
+            hasFlatFare.push($element);
+            cy.wrap(hasFlatFare).as('hasFlatFare');
+        }
+    });
+    cy.get('@hasFlatFare').then((hasFlatFare) => {
+        if (!hasFlatFare.toString()) {
```

Comment:
It looks like we only use the variable here - could it be a boolean?

Reply:
[3d27487](https://github.com/fares-data-build-tool/create-data/pull/649/commits/3d27487eba0e3b81baf7361f2988e0d13f6824c0)